amaranth_boards/tang_nano_9k.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+
+from amaranth.build import *
+from amaranth.vendor.gowin import *
+
+try:
+    from .resources import *
```

Comment:
This should only be `from .resources import *`; for testing you can install amaranth-boards in "editable mode".